Tests/GraphQLTransportWSTests-DataSync/Utils/TestMessenger.swift

```diff
@@ -1,6 +1,8 @@
+// Copyright (c) 2022 PassiveLogic, Inc.
+
 import Foundation
 
-@testable import GraphQLTransportWS
+@testable import GraphQLTransportWS_DataSync
```

Comment:
Wait, why do these `testable imports` use GraphQLTransportWS_DataSync with an underscore? I think it's supposed to have a dash...

Reply:
Thing that swift does. Any package names that have a dash have their import auto-replaced with an underscore. It will actually complain at you if you use a dash.

Reply:
Oh, huh. The more you know!